source/xrefprev

```diff
@@ -1,2764 +1 @@
-\glossaryentry{intro.scope@ {\memgloterm{intro.scope}}{\memglodesc{(\ref  {intro.scope})}} {\memgloref{}}|memjustarg}{1}
```

Comment:
The wholesale removal here is wrong.

This must be a copy of xrefindex.glo (NOT: xrefdelta.glo) at the point of C++23.

Reply:
Ah, I see, thanks!